clang/include/clang/Basic/Attr.td

```diff
@@ -1694,6 +1694,13 @@ def IntelFPGASimpleDualPort : Attr {
   let Documentation = [IntelFPGASimpleDualPortAttrDocs];
 }
 
+def IntelFPGAPipe : TypeAttr {
```

Comment:
Should we use "Intel" prefix here?
I understand that it's Intel extension, but I think Xilinx is also using this feature.
Can we use something more generic in the source code? E.g. `SYCLFPGAPipe`?

+@keryell, +@mkinsner

Reply:
Sure, let's use `SYCLFPGAPipe`